templates/etc/systemd/celeryd.service.j2

```diff
@@ -9,6 +9,7 @@ Group={{ django_system_group }}
 EnvironmentFile=-/etc/default/celeryd-{{ django_system_user }}
 WorkingDirectory={{ django_codebase_path }}
 RuntimeDirectory=celery-{{ django_system_user }}
+TimeoutStartSec= {{ django_celeryd_timeoutsec }}
```

Comment:
Nitpick: the empty space after equals bothers me because nothing else is like that in this file :)

Reply:
Resolved in latest commits